45 create job layout

sarah-tw opened the pull request.

Commit:
Remove node env param in package.json

sarah-tw committed:
Xiaojing&Zhaoyu: delete bootstrap

sarah-tw committed:
Xiaojing: add pure.less and basic layout

Commit:
ruoran: add validate commit message scripts

Commit:
ruoran: remove comments in script

winsonwq committed:
[fix]Qiu&Zekai: remove node-inspector grunt task

node-inspector grunt task will make the watch task
fail, not sure what's going on there. If you want 
to use it, please install `npm install -g node-inspector`.

winsonwq committed:
[fix]Qiu&Zekai: fix liverload bug

because we did not open 35729 port from vagrant so 
that liverload server is not accessible from outside.

sarah-tw committed:
Xiaojing&Zhaoyu: add create job static page.

sarah-tw committed:
Xiaojing&Zhaoyu: include fontawesome.

sarah-tw committed:
Xiaojing&Zhaoyu: init home page layout.

kiddkai committed:
[feature]zekai:add nconf to dependency

kiddkai committed:
[fix]zekai: move centre tpl from lib/home/views/index.jade to public

kiddkai committed:
[fix]zekai:bower.json wrong format

it caused build error

twers commented on public/styles/main.less:
```diff
@@ -1 +1,116 @@
+  color: #f1f1f2;
+  background-color: #333;
+  background-repeat: repeat-x;
+  background: -webkit-gradient(linear, 0% 0%, 0% 100%, from(#444), to(#333));
```
need to delete

twers commented on public/styles/main.less:
```diff
@@ -1 +1,116 @@
+      background: #a0c353;
+      color: #fefefe;
+    }
+    &.green{
```
change name to success or failure

twers commented on public/styles/main.less:
```diff
@@ -1 +1,116 @@
+  margin-top: 7px;
+  margin-right: 20px;
+}
+.create-job{
```
? general class

twers commented on public/styles/main.less:
```diff
@@ -1 +1,116 @@
+  margin-right: 20px;
+}
+.create-job{
+  width: 600px;
```
change to %

winsonwq committed:
[fix]Qiu: use bower to manage pure and font-awesome.

winsonwq committed:
[fix]Qiu: add debug line numder to compiled css file.

winsonwq committed:
[feature]Qiu: refactor the style structure

1. re-structure the style as per the component
2. add variables and mixins
3. clean the html structure

not finished. TBD
